shared/test/unit/mocks/mock_notification.js

```diff
@@ -12,6 +12,7 @@ function MockNotification(title, options) {
   this.icon = options.icon || undefined;
   this.body = options.body || undefined;
   this.tag = options.tag || undefined;
+  this.data = options.data || undefined;
```

Comment:
when I read this code, I think we should just remove all `|| undefined`... how about empty strings or `0` ?